dev/io.openliberty.io.opentelemetry.internal/bnd.bnd

```diff
@@ -0,0 +1,59 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include ~../cnf/resources/bnd/transform.props
+bVersion=1.0
+openTelemetryVersion=1.14.0
+
+Bundle-SymbolicName: io.openliberty.io.opentelemetry.internal; singleton:=true
+
+Import-Package: \
+  !io.opentelemetry.exporter.prometheus,\
```

Comment:
I don't think you need to exclude anything if you're not using a `*` in the imports